nerves_hub_client/client.py

```diff
@@ -19,9 +23,11 @@ class NervesHubAPI:
     """
     NervesHub API client
 
-    The canonical implementation is in the Elixir project [nerves_hub_user_api](https://github.com/nerves-hub/nerves_hub_user_api).
+    The canonical implementation is in the Elixir project
```

Comment:
While adding linting and updating docstrings, how do you feel about adding pydocstyle to the linters? I took a quick look at the output and it looks like its mostly adding periods to the docstrings and removing empty lines after function docstrings. Shouldn't be too much to get the repo passing.

Reply:
Sounds like a good idea